src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -35,9 +39,10 @@
 public class ModelManager extends ComponentManager implements Model {
     private static final Logger logger = LogsCenter.getLogger(ModelManager.class);
     private static final Predicate<ReadOnlyParcel> deliveredPredicate = p -> p.getStatus().equals(Status.COMPLETED);
+    private static final Index TAB_ALL_PARCELS = INDEX_FIRST_TAB;
+    private static final Index TAB_COMPLETED_PARCELS = INDEX_SECOND_TAB;
 
-    private static boolean selected = false;
-    private static Index prevIndex = Index.fromZeroBased(0);
+    private static Index tabIndex = Index.fromOneBased(1);
```

Comment:
why is this here? if it meant to be initialized, maybe put it in the constructor?